packages/edit-site/src/store/index.js

```diff
@@ -12,15 +12,20 @@ import * as actions from './actions';
 import * as selectors from './selectors';
 import { STORE_NAME } from './constants';
 
+export let store = {};
```

Comment:
Since `edit-site` is meant to be used only internally, I don't think this might be a problem, but if it's used outside of it, it might lead to unexpected errors if `store` is imported without calling `initialize` before.